src/input/Input.js

```diff
@@ -83,7 +85,13 @@ class Input extends React.Component {
     });
 
     return (
-      <View style={StyleSheet.flatten([styles.container, containerStyle])}>
+      <View
+        style={StyleSheet.flatten([
+          styles.container,
+          containerStyle,
+          disabled && styles.disabled,
```

Comment:
Let’s keep the styling only to input for now

Reply:
Alright, done!